architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,312 @@
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[[architecture-additional-concepts-dynamic-admission-webhooks]]
+== External Admission Webhooks
+
+External admission webhooks are HTTP callbacks that are intended to receive
+admission requests and do something with them. What an external admission
+webhook does is up to you, but there is an
+[interface](https://github.com/kubernetes/kubernetes/blob/v1.7.0-rc.1/pkg/apis/admission/v1alpha1/types.go)
+that it must adhere to so that it responds with whether or not the
+admission request should be allowed. 
+
+Unlike initializers or the plugin-style admission controllers, external
+admission webhooks are not allowed to mutate the admission request in any way.
+
+Because admission is a high security operation, the external admission webhooks
+must support TLS.
+
+A simple example use case for an external admission webhook is to do semantic validation
+of Kubernetes resources. Suppose that your infrastructure requires that all `Pod`
+resources have a common set of labels, and you do not want any `Pod` to be
+persisted to Kubernetes if those needs are not met. You could write your
+external admission webhook to do this validation and respond accordingly.
+
+Whenever a request comes in, the `GenericAdmissionWebhook` admission plugin will
+get the list of interested external admission webhooks from
+`externalAdmissionHookConfiguration` objects (explained below) and call them in
+parallel. If **all** of the external admission webhooks approve the admission
+request, the admission chain continues. If **any** of the external admission
+webhooks deny the admission request, the admission request will be denied, and
+the reason for doing so will be based on the _first_ external admission webhook
+denial reason. _This means if there is more than one external admission webhook
+that denied the admission request, only the first will be returned to the
+user._ If there is an error encountered when calling an external admission
+webhook, that request is ignored and will not be used to approve/deny the
+admission request.
+
+**Note:** The admission chain depends solely on the order of the
+`--admission-control` option passed to `kube-apiserver`.
+
+[[architecture-additional-concepts-dynamic-admission-webhooks-enable]]
+=== Enable External Admission Webhooks
+
+To enable the External Admission Webhooks feature:
+
+. Include `GenericAdmissionWebhook` in the `--admission-control` flag when
+  starting the apiserver. If you have multiple `kube-apiserver` replicas, all
+  should have the same flag setting.
+
+. Enable the dynamic admission controller registration API by adding
+  `admissionregistration.k8s.io/v1alpha1` to the `--runtime-config` flag passed
+  to `kube-apiserver`. 
++
+For example:
++
+----
+--runtime-config=admissionregistration.k8s.io/v1alpha1 
+----
++
+All replicas must have the same flag setting.
```

Comment:
replicas of what? The master?

Reply:
sttts So far, this document is a pull-down of a Kubernetes document that you added to the docs trello. I am going through trying to convert the source from K to OpenShift.
https://github.com/kubernetes/website/blob/b5a00984983a5494232bfed238e402a52482ccc1/docs/admin/extensible-admission-controllers.md#external-admission-webhooks